app/controllers/PaymentScheduleController.scala

```diff
@@ -74,8 +76,10 @@ class PaymentScheduleController @Inject() (
 
       case j: Journey.AfterSelectedPaymentPlan =>
         j match {
```

Comment:
This conditional is essentially so we audit for non email journeys at this point, but don't audit if it is an email journey - acceptance criteria asks for email info to be added to the event before submission (only applicable to email journeys).